app/domain/pension_summary.rb

```diff
@@ -0,0 +1,93 @@
+class PensionSummary
+  include ActiveModel::Model
+
+  # 'Steps' are all the topics involved in the summary,
+  # (including one that isn't offered as an option, 'final')
+  # Also this array specifies the order they're presented,
+  # after selection, which is different to the sequence in
+  # which they're offered up as options.
+  STEPS = %w(
+    leave_your_pot_untouched
+    get_a_guaranteed_income
+    get_an_adjustable_income
+    take_cash
+    take_whole
+    mix_your_options
+    how_my_pension_is_taxed
+    scams
+    how_my_pension_affects_my_benefits
+    getting_help_with_debt
+    taking_my_pension_if_im_ill
+    transferring_my_pension_to_another_provider
+    final
+  ).freeze
+
+  PRIMARY_OPTIONS = %w(
+    leave_your_pot_untouched
+    get_a_guaranteed_income
+    get_an_adjustable_income
+    take_cash
+    take_whole
+    mix_your_options
+  ).freeze
+
+  SECONDARY_OPTIONS = %w(
+    how_my_pension_affects_my_benefits
+    getting_help_with_debt
+    taking_my_pension_if_im_ill
+    transferring_my_pension_to_another_provider
+  ).freeze
+
+  COMPULSORY_OPTIONS = %w(
+    scams
+    how_my_pension_is_taxed
+  ).freeze
+
+  OPTIONS = [*PRIMARY_OPTIONS, *SECONDARY_OPTIONS, *COMPULSORY_OPTIONS].freeze
+
+  attr_accessor *STEPS
+
+  attr_accessor :current_step
+
+  def initialize(attributes)
```

Comment:
You don't use `attributes` so maybe just replace that arg with `*` and just call `super` bare?